common/models/user.js

```diff
@@ -752,10 +762,10 @@ module.exports = function(User) {
       {
         description: 'Logout a user with access token.',
         accepts: [
-          {arg: 'access_token', type: 'string', required: true, http: function(ctx) {
```

Comment:
The RFC says that it should be 401. Client frameworks expect 401 and subsequent WWW-Authenticate header to respond correctly. Also, other loopback APIs respond correctly with 401 when the token is missing.